site/category.xml

```diff
@@ -1,34 +1,23 @@
-		<category name="JBoss Tools jst Nightly Build Update Site"/>
-	</feature>
-        <feature url="features/org.jboss.tools.jst.jsdt.feature_0.0.0.jar" id="org.jboss.tools.jst.jsdt.feature" version="0.0.0">
-                <category name="JBoss Tools jst Nightly Build Update Site"/>
-        </feature>
-        <feature url="features/org.jboss.tools.jst.angularjs.feature_0.0.0.jar" id="org.jboss.tools.jst.angularjs.feature" version="0.0.0">
-                <category name="JBoss Tools jst Nightly Build Update Site"/>
-        </feature>
+  <description>To install these features, point Eclipse at this site.</description>
+  <category-def label="JBoss Tools JST Nightly Site" name="JBoss Tools JST Nightly Site"></category-def>
 
-	<feature url="features/org.jboss.tools.jst.test.feature_0.0.0.jar" id="org.jboss.tools.jst.test.feature" version="0.0.0">
-		<category name="JBoss Tools JST Tests Nightly Build Update Site"/>
-	</feature>
-	<!-- Sources -->
-	<feature url="features/org.jboss.tools.jst.feature.source_0.0.0.jar" id="org.jboss.tools.jst.feature.source" version="0.0.0">
-		<category name="JBoss Tools jst Nightly Build Update Site"/>
-	</feature>
-        <feature url="features/org.jboss.tools.jst.jsdt.feature.source_0.0.0.jar" id="org.jboss.tools.jst.jsdt.feature.source" version="0.0.0">
-                <category name="JBoss Tools jst Nightly Build Update Site"/>
-        </feature>
-        <feature url="features/org.jboss.tools.jst.angularjs.feature.source_0.0.0.jar" id="org.jboss.tools.jst.angularjs.feature.source" version="0.0.0">
-                <category name="JBoss Tools jst Nightly Build Update Site"/>
-        </feature>
-	<feature url="features/org.jboss.tools.jst.test.feature.source_0.0.0.jar" id="org.jboss.tools.jst.test.feature.source" version="0.0.0">
-		<category name="JBoss Tools JST Tests Nightly Build Update Site"/>
-	</feature>
+  <feature id="org.jboss.tools.jst.feature"><category name="JBoss Tools JST Nightly Site"/></feature>
+  <feature id="org.jboss.tools.jst.jsdt.feature"><category name="JBoss Tools JST Nightly Site"/></feature>
+  <feature id="org.jboss.tools.jst.angularjs.feature"><category name="JBoss Tools JST Nightly Site"/></feature>
+  <feature id="org.jboss.tools.jst.test.feature"><category name="JBoss Tools JST Nightly Site"/></feature>
+
+  <!-- Sources -->
+  <feature id="org.jboss.tools.jst.feature.source"><category name="JBoss Tools JST Nightly Site"/></feature>
+  <feature id="org.jboss.tools.jst.jsdt.feature.source"><category name="JBoss Tools JST Nightly Site"/></feature>
+	<feature id="org.jboss.tools.jst.angularjs.feature.source"><category name="JBoss Tools JST Nightly Site"/></feature>
+  <feature id="org.jboss.tools.jst.test.feature.source"><category name="JBoss Tools JST Nightly Site"/></feature>
+
+  <bundle id="minimal-json"/>
+  <feature id="tern-feature" version="0.7.0"><category name="JBoss Tools JST Nightly Site"/></feature>
```

Comment:
0.7.0 is not working here, it should be ether exact version with qualifier or 0.0.0 to pick up whatever is available from reactor.